apiserver/facades/agent/credentialvalidator/credentialvalidator.go

```diff
@@ -48,16 +49,15 @@ func internalNewCredentialValidatorAPI(backend Backend, resources facade.Resourc
 // changes to the given cloud credentials.
```

Comment:
Comment is plural but api call is single.
We should stick to convention and implement as a bulk call, even though we'll just be passing in one credential at the moment.